Mono.Debugging/Mono.Debugging.Client/AssemblyEventArgs.cs

```diff
@@ -30,13 +30,18 @@ namespace Mono.Debugging.Client
 {
 	public class AssemblyEventArgs : EventArgs
 	{
-		public AssemblyEventArgs (string location)
```

Comment:
This constructor can't be removed, it would be a public api break. You can keep it, and create an Assembly instance here with just the location.